src/lib/list/list.scss

```diff
@@ -27,7 +28,18 @@ $mat-dense-three-line-height: 76px;
 $mat-dense-multi-line-padding: 16px;
 $mat-dense-list-icon-size: 20px;
 
-$mat-list-item-inset-divider-offset: 72px;
```

Comment:
Removing this is a breaking change, even though it isn't being used anywhere.

Reply:
Since `mat-divider` hasn't been merged into the minor version yet, can we still remove it? Happy to keep it as a placeholder otherwise

Reply:
It could still be used from when the divider was in the list module.